build.sh

```diff
@@ -40,8 +40,8 @@
 # Build dependencies for scripts in bin/
 
 # Local directories for installing Python and R packages.
-R_LIB_DIR=`pwd`/work/rlibs
-PIP_TARGET_DIR=`pwd`/work/pylibs
+R_LIB_DIR=`pwd`/work/rlibs || exit $?
```

Comment:
These lines can't fail in a meaningful way, so no need to add ``|| exit $? here.